projects/storefrontlib/tsconfig.lib.json

```diff
@@ -13,7 +13,8 @@
     "types": [],
     "lib": ["esnext", "dom"],
     "paths": {
-      "@spartacus/core": ["dist/core"]
+      "@spartacus/core": ["dist/core"],
+      "@spartacus/incubator": ["dist/incubator"]
```

Comment:
I think the same applies to storefrontlib. It should never include an import to incubator lib.